### ISSUE TITLE
Allow customized chip appearances in autocomplete.

### ISSUE THREAD

**Issue by viridia** [WebPaperElements/paper-input-autocomplete-chips#11]
I'm using the autocomplete widget to gather a list of recipients for a post, I'd like for chips representing an individual to be a different color than chips representing a group of individuals. This could be easily done if the JSON data had a 'classes' field, representing additional CSS class names to add to the chip when it is created.

**Comment by patrinhani-ciandt** [comment]
Hi @viridia today it's possible due a property tag when you are adding the objects to the available items...
Here is a DEMO for that ..
http://webpaperelements.github.io/paper-input-autocomplete-chips/components/paper-input-autocomplete-chips/demo/index-custom-chips-styles.html

But i'm sure that we can think a better way to do that .. I was thinking to turn possible the set what will be the chip template ..

**Comment by viridia** [comment]
Yeah it would be nice to do it without ::shadow considering all of the dire warnings that Chrome gives about ::shadow and /deep/ being deprecated. Apparently custom properties is the way to go now, but there's no way to encode that in the json data.
